### PR TITLE
Updates HTTPS gatherer.

### PR DESCRIPTION
**Pull request by paullewis**
Now uses the Security domain instead of pinging the page's protocol.

### CODE REVIEW

File: src/lib/drivers/cri.js [hunk 1]
```diff
@@ -44,9 +44,13 @@ class CriDriver extends Driver {
         chromeRemoteInterface({port: port, chooseTab: tab}, chrome => {
           this._chrome = chrome;
           this.beginLogging();
-          this.enableRuntimeEvents().then(_ => {
-            resolve();
-          });
+
+          // Switch on events from domains we care about.
+          this.enableRuntimeEvents()
+              .then(_ => this.enableSecurityEvents())
```
Comment (paulirish) [inline src/lib/drivers/cri.js]: I think we want to enable this domain within our beforePageLoad gatherer. the events fire immediately, so we want the listener bound already.

Comment (samccone) [inline src/lib/drivers/cri.js]: 👍

Comment (paullewis) [inline src/lib/drivers/cri.js]: There are two gatherers that need the security events, so what's the order here? They both register their listeners, and I guess they can both enable the security events, but there's a disambiguation issue here: how do you know you've picked up the "most recent" securityChanged event here?

**Comment by paullewis** [comment]
Changed. PTAL.

**Comment by paulirish** [comment]
Yup. This is ace.

**Comment by brendankenny** [comment]
🔏 🔏

**Pull request merged**